features/step_definitions/generic_steps.rb

```diff
@@ -42,6 +42,14 @@
   puts "\n" + page.all("#main form")[(index.to_i-1)].native.inner_html
 end
 
+Then "I should see the {string} form" do |form_id|
+  assert page.has_xpath?("//form[@id='#{form_id}']")
```

Comment:
Would it work to do something like `expect(page).to have_xpath(...)` here, instead of using an assertion?

Reply:
+1, and I'd prefer `have_css` as well.